integration/internal/container/ops.go

```diff
@@ -46,6 +47,13 @@ func WithNetworkMode(mode string) func(*TestContainerConfig) {
 	}
 }
 
+// WithSysctls sets sysctl options for the container
+func WithSysctls(sysctls map[string]string) func(*TestContainerConfig) {
+	return func(c *TestContainerConfig) {
+		c.HostConfig.Sysctls = maps.Clone(sysctls)
```

Comment:
Ah, nice! We should have a look for other cases where `maps.Clone` could be applied. I expect there's probably quite some places where we copy basic maps (`map[string]string` and the likes)

Reply:
Yes, I was quite pleased with that! I guess hunting-for / replacing other uses should be a separate change.